css/properties/align-items.json

```
@@ -6,6 +6,7 @@
           "__compat": {
             "description": "Supported in Flex Layout",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/align-items",
+            "spec_url": "https://drafts.csswg.org/css-flexbox-1/#propdef-align-items",
```

```suggestion
            "spec_url": "https://drafts.csswg.org/css-flexbox-1/#align-items-property",
```

---

```suggestion
            "spec_url": "https://drafts.csswg.org/css-flexbox/#propdef-align-items",
```

---

That section defined two properties, but I guess that's OK.

---

I'll have to merge the suggestions into one in order to apply them:

```suggestion
            "spec_url": "https://drafts.csswg.org/css-flexbox/#align-items-property",
```

---

We need to decide whether to link to sections or directly to the definitions.

---

```suggestion
            "spec_url": "https://drafts.csswg.org/css-flexbox/#align-items-property",
```